tailwind.config.ts

```diff
@@ -0,0 +1,104 @@
+import type { Config } from 'tailwindcss';
+
+export default {
+  content: [],
+  theme: {
+    colors: {
+      green: {
+        100: '#EDF2EB',
+        200: '#C5E5B4',
+        300: '#C5E5B4',
+        400: '#84BA64',
+        500: '#5FA04E',
+        600: '#417E38',
+        700: '##2C682C',
+        800: '#2C682C',
+        900: '#1A3F1D',
+      },
+      neutral: {
+        100: '#F6F7F9',
+        200: '#E9EDF0',
+        300: '#D9E1E4',
+        400: '#CBD4D9',
+        500: '#B1BCC2',
+        600: '#929FA5',
+        700: '#6E7B83',
+        800: '#556066',
+        900: '#2C3437',
+      },
+      danger: {
+        100: '#FBF1F0',
+        200: '#FAD3D4',
+        300: '#FAB6B7',
+        400: '#FA8E8E',
+        500: '#F65354',
+        600: '#DE1A1B',
+        700: '#B80C0C',
+        800: '#900E0E',
+        900: '#661514',
+      },
+      warning: {
+        100: '#FDF3E7',
+        200: '#FAD9B0',
+        300: '#F5BC75',
+        400: '#E99C40',
+        500: '#D07912',
+        600: '#AE5F00',
+        700: '#8B4D04',
+        800: '#683D08',
+        900: '#4D2F0B',
+      },
+      info: {
+        100: '#E9F4FA',
+        200: '#BCE6FC',
+        300: '#8ED4F8',
+        400: '#52BAED',
+        500: '#229AD6',
+        600: '#0C7BB3',
+        700: '#066291',
+        800: '#074D71',
+        900: '#0A3953',
+      },
+      accent1: {
+        100: '#F7F1FB',
+        200: '#EAD9FB',
+        300: '#DBBDF9',
+        400: '#C79BF2',
+        500: '#AF74E8',
+        600: '#9756D6',
+        700: '#7D3CBE',
+        800: '#642B9E',
+        900: '#361B52',
+      },
+      accent2: {
+        100: '#FBF0F4',
+        200: '#FBD4E6',
+        300: '#FBB4D2',
+        400: '#F68BB7',
+        500: '#ED5393',
+        600: '#D6246E',
+        700: '#B01356',
+        800: '#8B1245',
+        900: '#411526',
+      },
+      withe: '#FFFFFF',
+      black: '#0D121C',
+    },
+    fontSize: {
+      xs: '0.75rem',
+      sm: '0.875rem',
+      base: '1rem',
+      lg: '1.125rem',
+      xl: '1.25rem',
+      '2xl': '1.5rem',
+      '3xl': '1.875rem',
+      '4xl': '2.25rem',
+      '5xl': '3rem',
+      '6xl': '3.75rem',
+      '7xl': '4.5rem',
+    },
```

Comment:
I didn't notice that the pull request was turned on when I was looking at the issues. @ovflowd @AugustinMauroy imho those updates can be good;

### Line height
You can also add `lineHeight`, also `xl` doesn't match with the default tailwind config. Also, the default config can be changed in the future.

Example usage of lineHeight;
```js
{
  xs: ['0.75rem', { lineHeight: '1rem' }],
  sm: ['0.875rem', { lineHeight: '1.25rem' }],
  base: ['1rem', { lineHeight: '1.5rem' }],
  lg: ['1.125rem', { lineHeight: '1.75rem' }],
  xl: ['1.25rem', { lineHeight: '1.875rem' }],
  '2xl': ['1.5rem', { lineHeight: '2rem' }],
  '3xl': ['1.875rem', { lineHeight: '2.25rem' }],
  '4xl': ['2.25rem', { lineHeight: '2.5rem' }],
  '5xl': ['3rem', { lineHeight: '1' }],
  '6xl': ['3.75rem', { lineHeight: '1' }],
  '7xl': ['4.5rem', { lineHeight: '1' }],
}
```
[Providing a default line height](https://tailwindcss.com/docs/font-size#providing-a-default-line-height)

### Font weight
You can match font weight with Figma.

Usage:
```js
fontWeight: {
  regular: '400',
  medium: '500',
  semibold: '600',
  bold: '700',
}
```
[Using custom font weight](https://tailwindcss.com/docs/font-weight#customizing-your-theme)

Reply:
Oh interesting I'll deep dive this proposal !

Reply:
@ovflowd do we need to specify `lineHeight` ?

Reply:
I genuinely don't know. I'd recommend reading Tailwind docs and experimenting. You probably need to test these.

@bmuenzenmeyer I genuinely think Storybooks should be part of this Issue/PR otherwise how can we validate these changes?

Reply:
I am fine with adding https://github.com/nodejs/nodejs.org/issues/5687 into this story, I just worry about too large a scope.  In this case it's probably fine as it feels like the natural way to validate.

Reply:
#5688 too